### PR TITLE
Minor codereview  of gpssh-exkeys

### PR DESCRIPTION
**Pull request by danielgustafsson**
The `:else` clause on the for loop is superfluous as the loop doesn't contain any break statement. Removing it will yield the same codepath but makes for improved readability.

This also removes an unused import (time) as well as fixes a set of typos.

### CODE REVIEW

File: gpMgmt/bin/gpssh-exkeys [hunk 1]
```diff
@@ -823,7 +822,7 @@ try:
     #    id_rsa{,.pub} files are copied from this host.  For
     #    existing hosts, the existing authorized_keys and known_hosts
     #    files from the existing host is merged with the files from
-    #    this nost
+    #    this host
     #
     print;
```
Comment (kainwen) [inline gpMgmt/bin/gpssh-exkeys]: first time to see a ';' in python code. The syntax is OK but strange and rare.

File: gpMgmt/bin/gpssh-exkeys [hunk 2]
```diff
@@ -726,9 +726,8 @@ try:
             GV.allHosts.remove(h)
             if h in GV.existingHosts: GV.existingHosts.remove(h)
             if h in GV.newHosts: GV.newHosts.remove(h)
-    else:
-        if len(badHosts):
-            sys.exit('[ERROR] cannot process one or more hosts')
+    if len(badHosts):
```
Comment (kainwen) [inline gpMgmt/bin/gpssh-exkeys]: `if len(SomeList)` -> `if (SomeList)`.

The latter is more pythonic and is suggested by pep8.

Comment (danielgustafsson) [reply gpMgmt/bin/gpssh-exkeys]: I don't disagree with this, but I've left this in place for now. The code has many instances of `len(list)` and I'd prefer if they were all fixed in one commit, but my Python skills are not really up to scratch for a bigger refactoring.

**Comment by danielgustafsson** [comment]
Pushed to master, thanks for review!

**Pull request closed without merging**